src/blockchain_utilities/blockchain_import.cpp

```diff
@@ -316,9 +316,9 @@ int import_from_file(cryptonote::core& core, const std::string& import_file_path
       MWARNING("WARNING: chunk_size " << chunk_size << " > BUFFER_SIZE " << BUFFER_SIZE);
       throw std::runtime_error("Aborting: chunk size exceeds buffer size");
     }
-    if (chunk_size > 100000)
+    if (chunk_size > 500000)
```

Comment:
Can we have a `#define CHUNK_MAXSIZE 500000` somewhere instead? Or maybe a CHUNK_WARNINGSIZE or something.

Reply:
Also, this same check is in bootstrap_file.cpp (but it's MDEBUG instead of MWARNING)

Reply:
I have no idea what that's for tbh. If I had to guess, I'd say it's a sanity check for "crazy values", which could possibly be removed...